.env

```diff
@@ -1 +1 @@
-NUSSKNACKER_VERSION=1.17.0
```

Comment:
It will be replaced by a published version after https://github.com/TouK/nussknacker/pull/6886 will be merged